english/light-hand/light-walls/in_my_belly/belly_ache.md

```diff
@@ -0,0 +1 @@
+You think you've solved your problem.  You have eaten your way out of the room.  But, you begin to feel an odd movement in your stomach.  You have not killed him; only moved him inside of you.  Your troubles have only just begun.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.